src/.env.template

```diff
@@ -55,7 +50,30 @@ USE_DEFAULT_IVS_STREAMS=true
 # Update to match your Evidently project name
 EVIDENTLY_PROJECT_NAME=retaildemostore
 
-# For orders service integration tests
+########################################################################
+## For Integration testing #############################################
+## These are default values for running locally with Docker Compose ####
+########################################################################
+# For products service integration tests
+
+# products service
+PRODUCTS_API_URL="http://localhost:8001"
+TEST_PRODUCT_ID="8bffb5fb-624f-48a8-a99f-b8e9c64bbe29"
+TEST_CATEGORY_NAME="tools"
+TEST_CATEGORY_ID="16"
+
+# users service
+USERS_API_URL="http://localhost:8002"
+TEST_USER_ID="1000"
+TEST_USERNAME="user1344"
+TEST_IDENTITY_ID="eu-west-1:12345678-1234-1234-1234-c777c9720775"
+TEST_PRIMARY_PERSONA="tools"
+TEST_AGE_RANGE="25-34"
+
+# orders service
 ORDERS_API_URL=http://localhost:8004
 TEST_ORDER_ID=1
-TEST_USERNAME=user1344
+TEST_USERNAME="user1344"
+
+# recommendation service
+RECOMMENDATIONS_API_URL="http://localhost:8005"
```

Comment:
This is just grouping everything together as we have many env vars now.

It took me a while to figure out all environment variables to get tests running. The workflow script can be very long to pass all of these and very error prone (one missing param will throw an error that is not obvious that it's missing an env var)

Thus, I made changes (you'll see later in this PR) to have default values for running all of this.